rules/3-Force-Management/2-Advanced-Force-Maintenance-custom-units.tex

```diff
@@ -0,0 +1,44 @@
+\item {\bfseries Custom Units}: Commanders may customize units by refitting an existing variant of a chassis.
+The unit retains the quirks of the chassis.
+The customized unit must use the same technology base as the variant of the chassis being customized and this technology base must be available to the force in this era.
+All equipment or weapons added must be available in the current era.
+The customized unit must be legal per \emph{BattleTech: TechManual} construction rules.
+
+League organizers set the highest customization level allowed in the league; each level includes all levels below it.
```

Comment:
Major internal changes such as changing the engine, gyro, or internal structure are not permitted.